src/data/examples/valid/MatrixEdgeList-001.yaml

```diff
@@ -5,7 +5,7 @@ edges:
   - subject: example:Node001
     predicate: "biolink:treats"
     object: example:Node002
-    knowledge_level: "high"
+    knowledge_level: "knowledge_assertion"
```

Comment:
updated, examples to use real values. I didn't add agent_type to the values in this PR, but I could (my main goal was predicate and category enums)